### PR TITLE
Closes #79 new release strategy vignette

### LINKED ISSUE #79: Documentation:  Create blurb in vignette about release labels
### Please select a category the issue is focused on?

Developer Guides

### Let us know where something needs a refresh or put your idea here!

Create blurb in vignette about release labels for all the repos

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: vignettes/release_strategy.Rmd [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# Introduction
+
+This article explains how we do package releases for `{admiral}` and across all our
+package extensions. This includes details around planned/scheduled releases, as well
+as hotfixes.
+
+# Release Schedule
+
+A release schedule is maintained on the [homepage](../index.html#release-schedule).
```
Comment (bms63) [inline vignettes/release_strategy.Rmd:28]: Should this get linked to the admiral core release schedule.  https://pharmaverse.github.io/admiral/#release-schedule

Comment (rossfarrugia) [reply vignettes/release_strategy.Rmd:28]: maybe - i spotted this too but left it as it was what was there before, and the link takes you to the admiraldev section which in turns links to the admiral section. want me to update it? is there any worry about circular dependencies somehow here, as admiraldev comes first and then admiral - i guess not as its just a link, but just thinking out loud... anyway, let me know which you prefer!

Comment (bms63) [reply vignettes/release_strategy.Rmd:28]: I think it is nicer for direct link then having to click a few more times to get to it?

Comment (bms63) [reply vignettes/release_strategy.Rmd:28]: oohhh i remember why i coded it like that.  the release was on `devel` branch on admiral core so I couldn't link anything at that time

**Pull request merged**